docs/content/configuration/csi_ug_config_create_vol_snapshotclass_topology.md

```diff
@@ -0,0 +1,49 @@
+# Creating a VolumeSnapshotClass with topology awareness
+
+When using the CSI Topology feature, different parameters must be taken into account when creating a VolumeSnapshotClass YAML file with specific `by_management_id` requirements. Use this information to help define a VolumeSnapshotClass that is topology aware and enables the creation and deletion of volume snapshots.
+
+**Note:** 
+  - For information and parameter definitions that are not related to topology awareness, be sure to see the information that is provided in [Creating a VolumeSnapshotClass](csi_ug_config_create_vol_snapshotclass.md), in addition to the current section.
+  
+  - IBM® FlashCopy® function is referred to as the more generic volume snapshots and cloning within this documentation set. Not all supported products use the FlashCopy function terminology.
+
+In order to enable creation and deletion of volume snapshots for your storage system, create a VolumeSnapshotClass YAML file, similar to the following `demo-volumesnapshotclass-config-secret.yaml`.
+
+When configuring the file, be sure to use the same array secret and array secret namespace as defined in [Creating a Secret with topology awareness](csi_ug_config_create_secret_topology.md).
+
+**Note:**
```

Comment:
why do we even have it here? we don't have it in the[ storageclass_topology](https://github.com/IBM/ibm-block-csi-driver/blob/1.7.0_documentation/docs/content/configuration/csi_ug_config_create_storageclasses_topology.md) config only in the [original one](https://github.com/IBM/ibm-block-csi-driver/blob/1.7.0_documentation/docs/content/configuration/csi_ug_config_create_storageclasses.md).

Reply:
I [agree](https://github.com/IBM/ibm-block-csi-driver/pull/360#discussion_r713168148)

Reply:
@ArbelNathan  if it's important for the customer to know it should be added into the places where it wasn't put in originally.
if it's not important we can remove.
it seems like we should add it into the places where it is missing. Part of writing the documentation is making sure that the customers do it right the first time and get all of the information that they need in order to prevent issues in the future.
let me know what you and @oriyarde  prefer and I'll do it across the different files.

Reply:
since we have this note:
```
  - For information and parameter definitions that are not related to topology awareness, be sure to see the information that is provided in [Creating a VolumeSnapshotClass](csi_ug_config_create_vol_snapshotclass.md), in addition to the current section.
```
and since we don't want to add more duplications, I still think it should be removed here

Reply:
we are not duplicate it in storageclass, I cant see why here we should..
the note to look at the definition of the files in the original object info page is sufficient in my opinion.

Reply:
After looking at the information and thinking about it some more - from a documentation point of view it should be in the other files to prevent errors on the part of the customer.
I've added it in to the other files. 
DONE

Reply:
IMO this just makes it worse, I don't think we need to duplicate every parameter note in every page with our parameters, just because of fear of errors.
would it help if we had a separate page for our parameters to which every "class" page would link?

Reply:
I agree with Ori and Arbel. The comment:
`  - For information and parameter definitions that are not related to topology awareness, be sure to see the information that is provided in [Creating a VolumeSnapshotClass](csi_ug_config_create_vol_snapshotclass.md), in addition to the current section.
`
is enough. We don't need to duplicate it (same apply to storage class)

Reply:
removed from storage class topology and snapshotclass topology
DONE

Reply:
I still see the [prefix notes](https://github.com/IBM/ibm-block-csi-driver/blob/3b2eae3cef526b6af3a1f0b48b1214f1cf109be3/docs/content/configuration/csi_ug_config_create_vol_snapshotclass_topology.md)

Reply:
DONE